webui/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildNpmPackage
+, nodejs
+, ...
+}:
+buildNpmPackage {
+  name = "HomestakerOS-frontend";
```

Comment:
Lower case pls

Reply:
I'll let this slide, this is only visible on Nix store and when building the package. The derivation has to be updated to use a proper version later anyway.